src/main/java/pl/javastart/voting/VotingResult.java

```diff
@@ -22,6 +41,18 @@ public void printResults() {
      * Nie zmieniaj sygnatury tej metody!
      */
     public void printVoteForVoter(String voterName) {
+        int index = getIndex(voterName);
+        Boolean vote = votingResult.get(index).getVote();
+        Vote voteCasted = new Vote(voterName, vote);
```

Comment:
Najpierw z głosu wyciągasz booleana, a później tworzysz ten głos na nowo, nie ma takiej potrzeby.
W code review pokazane jak to naprostować